docs/design/mockups/phase-4-homepage/round-5-newsgrid-structure-comparisons.html

```
@@ -0,0 +1,524 @@
+        </div>
+      </section>
+
+    </div>
+
+    <div class="footnote">
+      <p>
+        <strong>What's the same in every variant.</strong> 6 cards · `<code>tags[0]</code>` (or
+        articleType once #1470 ships) renders as the MonoLabel pill · publishedAt + author render
+        in the meta line · taped paper card primitive (TapedCard) · canonical press-down hover
+        applied at card level · NewsGrid header reuses Phase 2's `<SectionHeader>` (now thin
+        wrapper over `<EditorialHeading>` + `<MonoLabelRow>`).
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </monolabelrow></editorialheading></sectionheader> ], start tag match failed [ <MonoLabelRow> ] on line 514.

(tag-pair)